### PR TITLE
added parameter count to CLICK

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
If you use buttons as functions, you can use this parameter to click them multiple times. This makes it possible to read the count of clicks from a variable.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3048/pr-test (or any other room on that server)

File: client/js/widgets/button.js [hunk 1]
```diff
@@ -85,10 +85,11 @@ export class Button extends Widget {
 
 
       if(a.func == 'CLICK') {
-        setDefaults(a, { collection: 'DEFAULT' });
+        setDefaults(a, { collection: 'DEFAULT', count: 1 });
         if(isValidCollection(a.collection))
           for(const w of collections[a.collection])
-            w.click();
+            for(let i=0; i<a.count; ++i)
+              w.click();
```
Comment (UltimateGeek) [inline client/js/widgets/button.js:92]: I would rather click all items in the collection, and then repeat until count has been reached.

**Review by UltimateGeek — COMMENTED** [review]
* [ ] update wiki Button page once PR is merged

File: client/js/widgets/button.js (hunk 1 above)
Comment (ArnoldSmith86) [reply client/js/widgets/button.js:92]: I didn't even think about that. But I agree.

* [ ] change click order

**Review by UltimateGeek — APPROVED** [review]

**Comment by RaphaelAlvez** [comment]
@larkob I'm approving this one. And moving it in the wiki project into to-do

**Review by RaphaelAlvez — APPROVED** [review]
Approved concept and code

**Pull request merged**